test/javascripts/test_helper.js

```diff
@@ -1,7 +1,7 @@
 /*global document, sinon, QUnit, Logster */
 
 //= require env
-//= require ../../app/assets/javascripts/preload_store
```

Comment:
This is really odd but base on what I saw in the source code, `require` requires the path to have the same root path as the current file.